docs/envlist.rst

```diff
@@ -63,10 +63,10 @@ For example, see the following ``.travis.yml`` and ``tox.ini``:
     language: python
     python:
       - "2.7"
-      - "3.4"
+      - "3.7"
     env:
-      - DJANGO="1.7"
-      - DJANGO="1.8"
+      - DJANGO="1.11"
+      - DJANGO="2.1"
```

Comment:
I'm not keen on doing it like this, because Django 2.1 doesn't actually work with Python 2.7. You could use lower versions of Django, but I'm inclined to just use the last two releases of Django with the latest 2 releases of Python for the example matrix. So Python 3.7 and 3.8 with Django 2.1 and 2.2.

Reply:
Done, I used py36 and py37 as Django 2.x will not 'officially' support py38

Reply:
Makes sense, thank you!